web/src/css-loader.test.js

```diff
@@ -0,0 +1,10 @@
+const pkg = require('../package.json');
+
+// This test can be deleted entirely when the issue is fixed and the library
+// can safely be upgraded.
+
+describe('make sure css-loader is fixed before updating', () => {
+  it('We cannot update css-loader until this issue is fixed: https://github.com/webpack-contrib/css-loader/issues/1039', () => {
+    expect(pkg.devDependencies['css-loader']).toEqual('3.2.1');
```

Comment:
This really is a brilliant way of ensuring that this doesn't accidentally get updated.